modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/InteractListener.java

```diff
@@ -17,21 +17,25 @@
 
 package net.countercraft.movecraft.listener;
 
-import net.countercraft.movecraft.utils.MathUtils;
-import net.countercraft.movecraft.craft.Craft;
 import net.countercraft.movecraft.config.Settings;
+import net.countercraft.movecraft.craft.Craft;
 import net.countercraft.movecraft.craft.CraftManager;
 import net.countercraft.movecraft.localisation.I18nSupport;
+import net.countercraft.movecraft.utils.MathUtils;
 import org.bukkit.Material;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
 import org.bukkit.event.block.Action;
 import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.util.Vector;
 
 import java.util.HashMap;
 import java.util.Map;
 
+import static java.lang.Math.abs;
+import static java.lang.Math.rint;
```

Comment:
I'd like to hear an opinion from @cccm5 on this, but I believe we have stayed away from static importing math functions for a reason.

Reply:
@cccm5 ?